### ISSUE TITLE
Problem with variation_weight

### ISSUE THREAD

**Issue by yloquen** [chrisgoringe/cg-noise#13]
The variation_weight only works for me when it's a small deviation from 0 or 1 - for example 0.05 or 0.95, so using mostly one of the seeds. For values like 0.5 I get entirely blank image as output. Any idea what could be causing this? It reminds me of generating with an empty latent with a denoise set below 1.00.
I looked at the tensors generated in prepare_mixed_noise and the numbers seem correctly interpolated.
P.S. Tested with SDXL1safetensors and SD2.1
![Capture](https://github.com/chrisgoringe/cg-noise/assets/4171176/05213b14-2cd4-40cc-af87-20f3841927fc)
![Capture2](https://github.com/chrisgoringe/cg-noise/assets/4171176/02c57caf-85f1-4f4b-9aba-91dff255815e)

**Comment by chrisgoringe** [comment]
Interesting. Could you link your complete workflow?

**Comment by yloquen** [comment]
[workflow(2).json](https://github.com/chrisgoringe/cg-noise/files/13260075/workflow.2.json)
Here it is
Edit: When I load the examples from the repo, I get some strange settings - for example the cfg is NaN, the scheduler is 1 instead of string, etc. maybe there are changes in Comfy, I updated it to latest version. 
When I fix those settings, I still get similar results at 0.5 weight.
Edit2: After a bit more experimenting, I found that the issue arises from the choice of sampler/scheduler. When I select dpmpp_sde with karras, I get results similar to the examples.
